internal/events/release_succeeded.go

```diff
@@ -0,0 +1,35 @@
+package events
+
+import (
+	"encoding/json"
+
+	"github.com/lunarway/release-manager/internal/broker"
+)
+
+type ReleaseSucceeded struct {
+	Name          string `json:"name,omitempty"`
+	Namespace     string `json:"namespace,omitempty"`
+	ResourceType  string `json:"resourceType,omitempty"`
+	AvailablePods int32  `json:"availablePods,omitempty"`
+	DesiredPods   int32  `json:"desiredPods,omitempty"`
+	ArtifactID    string `json:"artifactID,omitempty"`
```

Comment:
is artifactID also capitalized `ID` in the json as well?

Reply:
Great catch 🤚